production/ksonnet/loki/config.libsonnet

```diff
@@ -2,7 +2,7 @@
   _config+: {
     namespace: error 'must define namespace',
     cluster: error 'must define cluster',
-    http_listen_port: 3100,
+    http_listen_port: 80,
```

Comment:
I wonder if we should leave this as 3100? and instead override it for our environments?

Reply:
No strong opinion ! I can make that change.